targets/LiteOS-stm32f4/source/main.c

```diff
@@ -0,0 +1,150 @@
+#include "los_sys.h"
+#include "los_tick.h"
+#include "los_task.ph"
+#include "los_config.h"
+
+#include "los_bsp_led.h"
+#include "los_bsp_key.h"
+#include "los_bsp_uart.h"
+
+#include "stm32f429i_discovery.h"
+//#include "stm32f4xx_conf.h"
+
+#include <string.h>
+//#include <stdio.h>
+
+#include "jerryscript.h"
+#include "jerry_run.h"
+
+int _write(int file, char *ptr, int len)
+{
+#if 0
+	int idx;
+	for(idx = 0; idx < len; idx++)
+	{
+		LOS_EvbUartWriteByte((char)(*ptr));
+		ptr++;
+	}
+	return len;
+#else
+	LOS_EvbUartWriteStr(ptr);
+	return len;
+
+#endif
```

Comment:
This file is out-of-style as long as JerryScript coding guidelines go, and is of mixed style. Please, use consistent style for: indentation and whitespacing in general, function signatures, comments, etc.